src/components/CollaborationBar/CollaborationBar.tsx

```diff
@@ -15,12 +15,17 @@
  */
 
 import { useTranslation } from 'react-i18next';
+import { usePresentationMode } from '../../state';
 import { Toolbar } from '../common/Toolbar';
 import { Collaborators } from './Collaborators';
 import { ShowCollaboratorsCursorsToggle } from './ShowCollaboratorsCursorsToggle';
 
 export function CollaborationBar() {
   const { t } = useTranslation();
+  const { state: presentationState } = usePresentationMode();
+  const isCollaboratorsCursorsActive =
+    presentationState.type === 'idle' || presentationState.isEditMode;
```

Comment:
This variable is more about that we can edit whiteboard or the current slide and not about showing cursors. I have commented this because other changes in this PR use the same condition but name it in terms of edit that sounds more correct.